OpenRA.Game/Graphics/SpriteRenderer.cs

```diff
@@ -160,9 +160,9 @@ public void SetViewportParams(Size screen, float depthScale, float depthOffset,
 			shader.SetVec("Scroll", scroll.X, scroll.Y, scroll.Y);
 			shader.SetVec("r1",
 				zoom * 2f / screen.Width,
-				-zoom * 2f / screen.Height,
+				zoom * 2f / screen.Height,
 				-depthScale * zoom / screen.Height);
-			shader.SetVec("r2", -1, 1, 1 - depthOffset);
+			shader.SetVec("r2", -1, -1, 1 - depthOffset);
```

Comment:
Before anyone asks: this change and the `EnableScissor` rename are removing the hardcoded y coordinate flip that used to be baked in to the rendering to work around OpenGL's origin being the bottom left of the window. We now preserve coordinates through the frame buffer(s) and do the flip at the end by rendering the screen quad upside down.

Reply:
OK.